docs/api.md

```diff
@@ -1951,6 +2008,39 @@ You can fully delete a link by issuing `DELETE` to its resource. This will remov
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+
+## Dataset Properties [/v1]
+
+_(introduced: version 2022.3)_
+
+Datasets are created and updated through Forms. Dataset-related Forms follow a new version of the [ODK XForms specification](https://getodk.github.io/xforms-spec) that allow them to define a Dataset and a mapping of Form Fields to Dataset Properties. Submissions from such a Form can create Entities within the Dataset defined in the Form. 
```

Comment:
"new version..." -> "`[the entities sub-spec](https://getodk.github.io/xforms-spec/entities)` of the ODK XForms specification